pkgs/development/libraries/freeimage/unbundle.diff

```diff
@@ -0,0 +1,611 @@
+diff --git a/Makefile.fip b/Makefile.fip
+index 660a026..b93bba8 100644
+--- a/Makefile.fip
++++ b/Makefile.fip
+@@ -4,39 +4,27 @@
+ include fipMakefile.srcs
+ 
+ # General configuration variables:
+-DESTDIR ?= /
+-INCDIR ?= $(DESTDIR)/usr/include
+-INSTALLDIR ?= $(DESTDIR)/usr/lib
++INCDIR ?= $(out)/include
++INSTALLDIR ?= $(out)/lib
```

Comment:
Should work to do
```nix
makeFlags = [
  #"DESTDIR=${placeholder "out"}"
  "INCDIR=${placeholder "out"}/include"
  "INSTALLDIR=${placeholder "out"}/lib"
];
```

Reply:
I decided not to do this since this wouldn't work for all targets. In Makefile.osx, `INCDIR` etc. are defined not using `?=` but just `=`, so the patch is necessary anyway, no?

Reply:
@L-as Yes, I saw that too. IIRC it also overrides `=` (but not `:=`).

Reply:
I couldn't make it work with makeFlags, but it works if I export it in preInstall.